exec/Chainweb/Env.hs

```diff
@@ -258,8 +261,8 @@ bfArgsP = BackfillArgs
   <$> delayP
   <*> optional (option auto (long "chunk-size" <> metavar "CHUNK_SIZE" <> help "Number of transactions to query at a time"))
 
-gapArgsP :: Parser GapArgs
-gapArgsP = GapArgs
+fillArgsP :: Parser FillArgs
+fillArgsP = FillArgs
   <$> delayP
   <*> flag False True (long "disable-indexes" <> short 'd' <> help "Disable indexes on tables while filling on gaps.")
```

Comment:
Maybe update the comment here as well (maybe not use the words gaps).

Reply:
Ahh, good catch